src/EnergyPlus/DataHVACGlobals.hh

```diff
@@ -320,6 +320,7 @@ namespace DataHVACGlobals {
     extern Real64 DXElecCoolingPower;       // Electric power consumed by DX cooling coil last DX simulation
     extern Real64 DXElecHeatingPower;       // Electric power consumed by DX heating coil last DX simulation
     extern Real64 ElecHeatingCoilPower;     // Electric power consumed by electric heating coil
+    extern Real64 SuppHeatingCoilPower;     // Electric power consumed by electric supplemental heating coil
```

Comment:
While having another global isn't great, it fits the existing pattern, so ok.

Reply:
Should these be struct vars instead?

Reply:
No. they can't be struct vars because they are set in the coil models and used in the parent models. So what other methods could eliminate the globals here?

Reply:
A new class I suppose, that is inherited by all who need those vars.